packages/x-data-grid/src/constants/localeTextConstants.ts

```diff
@@ -42,6 +42,9 @@ export const GRID_DEFAULT_LOCALE_TEXT: GridLocaleText = {
   // Toolbar pivot button
   toolbarPivot: 'Pivot',
 
+  // Toolbar pivot button
+  toolbarCharts: 'Configure charts',
```

Comment:
Would prefer to keep this label simple, like the other toolbar items:

```suggestion
  toolbarCharts: 'Charts',
```